### ISSUE TITLE
(UI updates) Remove the extra semicolon (;) on each of the select tabs

### ISSUE THREAD

**Issue by KC900201** [Buzzpy/Dev-Encyclopedia#135]
Taking a look at the current home page's UI layout, it would look seamer and cleaner if it the extra semicolon at the end of each wording is removed as it seems a little bit redundant like removing the "The" from the original **The Facebook** title before it was rebranded as just **Facebook**
![image](https://github.com/user-attachments/assets/19af800d-23e6-4b93-bffc-2c45136177ba)

I would like your opinion on this before we can escalate this to an improvement.
cc: @Buzzpy, @RayMathew.

**Comment by Buzzpy** [comment]
Hello again!
That's a great idea! I too felt that the semicolon on the title "Dev Encyclopedia" needs to be removed, but I doubt if the buttons's semicolons should be removed. 

I kind of love the look of it, with a touch of coding on the website.

**Comment by RayMathew** [comment]
Agreed, I too felt that the semicolon adds a unique style to the site. But if we're talking about deleting something I would delete the ones on the buttons 😂 No right answer I guess. Up to you Buzzpy.

**Comment by Buzzpy** [comment]
Got it, let's remove the buttons then! I will assign you to this.

**Comment by KC900201** [comment]
Hi @Buzzpy,

A new PR #138 is created for this task. Kindly review at your convenience.

cc: @RayMathew

**Comment by Buzzpy** [comment]
I reviewed it and merged it! The `develop` branch will be synced to the `master` soon.

I'll close the issue now. Thanks for the contribution!
